CHANGELOG.md

```diff
@@ -2,12 +2,7 @@
 
 ### Thanks
 
-Thanks to the following projects which we fully rely on to provide some of
-our features:
-
--   [Python Extension](https://marketplace.visualstudio.com/items?itemName=ms-python.python)
-
-Also thanks to the various projects we provide integrations with which help
```

Comment:
Removed section where we thank python extension, not required & not true, we don't rely on that for anything in renderers.